psi4/src/psi4/libmints/electricfield.h

```diff
@@ -47,17 +47,6 @@ class Molecule;
  *  Use an IntegralFactory to create this object.
```

Comment:
I cannot comment on the precise line because apparently GitHub doesn't allow me...
Could you maybe remove `#include "psi4/libmints/osrecur.h"` where it's not needed anymore?